code/index.html

```diff
@@ -16,29 +16,66 @@
 
       <img
         class="guess-who-icon"
-        src="images/guess-who-bubble.png"
+        src="images/reed.svg"
         alt="Guess Who"
       />
 
-      <h1>Does the person have</h1>
+      <h1>Which fish <br> is hiding <br> in the reed?</h1>
 
       <select id="questions">
-        <optgroup label="hair">
-          <option value="brown">brown hair</option>
-          <option value="yellow">yellow hair</option>
+        
+        <option selected="true" disabled="disabled">Does the fish...</option>    
+
+        <optgroup id="optscale" class="options" label="scale">
+        
+          
+          <option value="black">black</option>
+          <option value="yellow">yellow</option>
+          <!-- Add the rest of hair colors as options  -->
+          <option value="green">green</option>
+          <option value="orange">orange</option>
+          <option value="blue">blue</option>
+          <option value="red">red</option>
+
           <!-- Add the rest of hair colors as options  -->
         </optgroup>
-        <optgroup label="eyes">
-          <option value="hidden">hidden eyes</option>
+
+
+        <optgroup id="optexpression" class="options" label="expression">
+
+          <option value="happy">happy</option>
+          <!-- Add the rest of eye colors as options  -->
+          <option value="sad">sad</option>
+          <option value="astonished">astonished</option>
+          <option value="expressionless">expressionless</option>
+          <option value="sneaky">sneaky</option> 
+
+
+
           <!-- Add the rest of eye colors as options  -->
         </optgroup>
-        <optgroup label="accessories">
-          <option value="glasses">glasses</option>
+
+
+        <optgroup id="optshape" class="options" label="shape">
+
+          <option value="round">round</option>
+          <!-- Add the other accessory option here. (hat)  -->
+          <option value="long">long</option>
+          <option value="short">short</option>
+          <option value="short">average</option>
           <!-- Add the other accessory option here. (hat)  -->
         </optgroup>
-        <optgroup label="other">
-          <option value="smoker">a smoking habit</option>
+
+        <optgroup id="optage" class="options" label="age">
+
+          <option value="young">young</option>
+          <!-- Add the other accessory option here. (hat)  -->
+          <option value="middleage">middleage</option>
+          <option value="old">old as the sea</option>
+          <!-- Add the other accessory option here. (hat)  -->
```

Comment:
I would suggest removing the comments on lines 34, 40, 55, and 66 as they are no longer relevant.